docs/source/authentication.rst

```diff
@@ -4,17 +4,36 @@
 Authentication
 **************
 
-Set up authentication keys
-==========================
-You can authenticate with |company| on a user basis by registering a public key, or on a specific project or dataset basis by using an |company|-generated API key tied to that resource.
+Types of authentication
+========================
+You can authenticate with |company| on a user basis by registering an :xref:`ssh_public_key_authentication` (i.e. "public key" in this document), or on a specific project or dataset basis by using an |company|-generated API key tied to that resource.
```

Comment:
We ant to hide the use of API keys I think as we just want to tell people to use SSH keys?

Reply:
I see below you have emphasised the use of ssh keys. That might be good enough. @justin-cord what do you think?

Reply:
I think we should revisit the decision to completely hide this for now because
* The way we have structured the docs now clients know how to use the SSH key and are encouraged to use it in multiple places. This adds the value that we initially wanted to create already.
* If someone is not a project/dataset/organisation admin, they will still be able to use the API key to do some operations.
* If we want to have fine grained access control we need to do this via API keys for now - that's why we'd want to still support them.

Unless we really think that most clients would still be confused about having the API key access, I'd personally keep the documentation in because of the additional use cases. Seems like we can have more value for less work (of deprecating) that way.

Reply:
quick comment to say my above comment seemed to mirror Riz's comment, catching up on the above...